core/src/test/resources/channels/channel-with-wildcard-group.yaml

```diff
@@ -0,0 +1,12 @@
+schemaVersion: "1.0.0"
```

Comment:
Is this file used anywhere in the testsuite?

Reply:
No, I implemented the test differently in the end. I'll remove this